website/OpenEXRFileLayout.rst

```diff
@@ -991,177 +991,858 @@ pixels. The image has two channels: G, of type ``HALF``, and Z, of type
+  +-----------+------------+Width        |
+  |72         |r           |             |
+  +-----------+------------+             |
+  |65         |e           |             |
+  +-----------+------------+             |
+  |65         |e           |             |
+  +-----------+------------+             |
+  |6e         |n           |             |
+  +-----------+------------+             |
+  |57         |W           |             |
+  +-----------+------------+             |
+  |69         |i           |             |
+  +-----------+------------+             |
+  |6e         |n           |             |
+  +-----------+------------+             |
+  |64         |d           |             |
+  +-----------+------------+             |
+  |6f         |o           |             |
+  +-----------+------------+             |
+  |77         |w           |             |
+  +-----------+------------+             |
+  |57         |W           |             |
+  +-----------+------------+             |
+  |69         |i           |             |
+  +-----------+------------+             |
+  |64         |d           |             |
+  +-----------+------------+             |
+  |74         |t           |             |
+  +-----------+------------+             |
+  |68         |h           |             |
+  +-----------+------------+             |
+  |00         |\\0         |             |
+  +-----------+------------+-------------+
+  |66         |f           |attribute    |
+  +-----------+------------+type         |
+  |6c         |l           |             |
+  +-----------+------------+             |
+  |6f         |o           |             |
+  +-----------+------------+             |
+  |61         |a           |             |
+  +-----------+------------+             |
+  |74         |t           |             |
+  +-----------+------------+             |
+  |00         |\\0         |             |
+  +-----------+------------+-------------+
+  |04         |4           |size         |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+------------+-------------+
+  |00         |1.0         |attribute    |
+  +-----------+            |value        |
+  |00         |            |             |
+  +-----------+            |             |
+  |80         |            |             |
+  +-----------+            |             |
+  |3f         |            |             |
+  +-----------+------------+-------------+
+  |00         |\\0 - end of header       |
+  +-----------+------------+-------------+
+  |End of header                         |
+  +-----------+------------+-------------+
+  |3f         |319         |offset of    |
+  +-----------+            |scan line 0  |
+  |01         |            |start of     |
```

Comment:
The `start of scan line offset table` should span all columns, just below `End of header`, since it's an indication of the coming section of the file. Same with `end of scan line offset table`